his/core/as-400-connection-summary1.md

```diff
@@ -8,13 +8,13 @@ ms.reviewer: ""
 ms.suite: ""
 ms.topic: "article"
 ---
-# AS/400 Connection Summary
+# IBM System i Connection Summary
 In the peer-oriented SNA network model, all computers on the network can communicate directly with each other. Advanced Peer-to-Peer Networking (APPN) is the architecture developed by IBM that enables distributed data processing. APPN defines how components communicate with each other, and the level of network-related services, like routing sessions, that are supplied by each computer in the network.  
   
- ![Image that shows Host Integration Server connecting to an AS/400 with several connection methods.](../core/media/dep07.gif "dep07")  
-Diagram showing Host Integration Server connecting to an AS/400 with several connection methods  
+ ![Image that shows Host Integration Server connecting to an IBM System i with several connection methods.](../core/media/dep07.gif "dep07")  
+Diagram showing Host Integration Server connecting to an IBM System i with several connection methods  
```

Comment:
Caption should appear preceding the image:

`The following diagram shows a Host Integration Server connection to an IBM System i using several connection methods:`